### PR TITLE
Transformer event sanity

### PR DESCRIPTION
**Pull request by utsabc**
## Transformer Event Sanity

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: __tests__/data/sanity/active_campaign_output.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+
```
Comment (utsabc) [inline __tests__/data/sanity/active_campaign_output.json:2]: Under the sanity folder, we will have each of the sanity output files for each of the destinations.

File: __tests__/data/sanity/active_campaign_output.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+}
```
Comment (utsabc) [inline __tests__/data/sanity/active_campaign_output.json:3]: In the sanity folder, we will have all the output JSON for each of the destination's sanity Test Case

File: __tests__/data/sanity/sanity_input.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "messages": [
+        {},
+        {}
+    ],
+    "config": {
+        "active_campaign": {}
+    }
+}
```
Comment (utsabc) [inline __tests__/data/sanity/sanity_input.json:9]: For each of the destination we will have the following sanity input containing an array of input events under messages and all the configs with respective destination keys.

File: __tests__/sanity.test.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        const output = await transformer.process(event);
+        expect(output).toEqual(expectedData);
+      } catch (error) {
+        expect(error.message).toEqual(expectedData.error);
+      }
+    });
+  });
+});
+
+console.log(output);
```
Comment (utsabc) [inline __tests__/sanity.test.js:41]: Here we are iterating and getting all the directory names, using each of the directory names under destination we are using them to generate transformer and use the sanity input to get data.

Comment (ruchiramoitra) [reply __tests__/sanity.test.js:41]: please write these comments in the code itself.

**Review by arnab-p — APPROVED** [review]

**Pull request merged**